operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -315,21 +315,60 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
       Generating type static...
       ```
 
-5. Resubmit the Kubernetes Job that runs the Keycloak localization tool.
+5. Re-apply the cray-keycloak-users-localize Helm chart with the updated customizations.yaml file.
+
+    1.  Determine the cray-keycloak-users-localize chart version that is currently deployed.
+
+        ```bash
+        ncn-m001# helm ls -A -a | grep cray-keycloak-users-localize
+        ```
+
+    2.  Create a manifest file that will be used to reapply the same chart version.
+
+        ```bash
+        ncn-m001# cat << EOF > ./cray-keycloak-users-localize-manifest.yaml
+        apiVersion: manifests/v1beta1
+        metadata:
+        name: reapply-cray-keycloak-users-localize
+        spec:
+        charts:
+        - name: cray-keycloak-users-localize
+            namespace: services
+            values:
+            imagesHost: dtr.dev.cray.com
+            version: 0.12.2
+        EOF
+        ```
+
+    3. Determine the CSM_RELEASE version that is currently running and set an environment variable.
+
+        For example:
 
-    1.  Re-apply the cray-keycloak-users-localize Helm chart with the updated customizations.yaml file.
-   
         ```bash
-        ncn-m001# kubectl get job -n services -l app.kubernetes.io/name=cray-keycloak-users-localize \
-        -ojson | jq '.items[0]' > keycloak-users-localize-job.json 
-        
-        ncn-m001# cat keycloak-users-localize-job.json | jq 'del(.spec.selector)' | \
-        jq 'del(.spec.template.metadata.labels)' | kubectl replace --force -f -
-        job.batch "keycloak-users-localize-1" deleted     
-        job.batch/keycloak-users-localize-1 replaced
+        ncn-m001# CSM_RELEASE=1.10.54
         ```
 
-    2. Watch the pod to check the status of the job.
+    4. Mount the PITDATA so that helm charts are available for the re-install (it might already be mounted).
+
+        ```bash
+        ncn-m001# mkdir -pv /mnt/pitdata
+        ncn-m001# mount -L PITDATA /mnt/pitdata
+        ```
+
+    5. Uninstall the current cray-keycloak-users-localize chart.
+
+        ```bash
+        ncn-m001# helm del cray-keycloak-users-localize -n services
+        ```
+
+    6. Reapply the cray-keycloak-users-localize chart based on the CSM_RELEASE.
+
+        ```bash
+        ncn-m001# loftsman ship --manifest-path ./cray-keycloak-users-localize-manifest.yaml \
```

Comment:
how does this get the current customizations.yaml?

Reply:
You need a call to manifestgen to populate the deployment manifest with data from customizations.yaml

`manifestgen -i cray-keycloak-users-localize-manifest.yaml -c customizations.yaml -o deploy.yaml`

You would then use the resulting deploy.yaml for the `loftsman ship` command.